src/test/regress/Makefile

```diff
@@ -209,6 +210,9 @@ check-failure-base: all
 check-pg-upgrade:
 	$(pg_upgrade_check) --old-bindir=$(old-bindir) --new-bindir=$(new-bindir) --pgxsdir=$(pgxsdir)
 
+check-custom-citus:
```

Comment:
we might find a more expressive test name

Reply:
I would prefer shorter names for make targets as we write them manually from the terminal a lot.

Some suggestions I have:
- check-sql-correctness
- check-core-sql
- check-different-configs

All of them seems to be missing something, so maybe we can think a bit more

Reply:
Onder suggested `check-arbitrary-configs`, which seems like the best option among the ones we could come up with